go.mod

```diff
@@ -44,9 +44,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20190130013101-742f2714c145+incompatible
-	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
```

Comment:
These changes should not be in this PR. They are a result of not having the `e` submodule checked out.

Reply:
woops good catch!

Reply:
@zmb3 fixed and squashed